jericho/README.md

```diff
@@ -3,9 +3,83 @@
 jericho is the central server for book metadata and review information
 
 ### stack
+
 - Server language: Python 3
-    - Package management: [Poetry](https://python-poetry.org/)
-    - Framework: [FastAPI](https://fastapi.tiangolo.com/)
-    - ORM: [SQLModel (SQLAlchemy + Pydantic)](https://sqlmodel.tiangolo.com/)
-    - Migrations: [Alembic](https://alembic.sqlalchemy.org/en/latest/)
+  - Package management: [Poetry](https://python-poetry.org/)
+  - Framework: [FastAPI](https://fastapi.tiangolo.com/)
+  - ORM: [SQLModel (SQLAlchemy + Pydantic)](https://sqlmodel.tiangolo.com/)
+  - Migrations: [Alembic](https://alembic.sqlalchemy.org/en/latest/)
 - Relational database: PostgreSQL 15
+
+## Setup
```

Comment:
@ardenchew happy to move this to a separate `SETUP.md` or `DEVELOPMENT.md` file etc if that's cleaner.